### PR TITLE
Bring back buttons, fix context keys

### LINKED ISSUE #232966: Terminal chat cmd/alt+enter doesn't work on mac
cmd/alt+enter doesn't work in this state on mac:

Image

cmd+enter:

```
2024-11-04 05:11:44.840 [info] [KeybindingService]: / Soft dispatching keyboard event
2024-11-04 05:11:44.841 [info] [KeybindingService]: \ Keyboard event cannot be dispatched
2024-11-04 05:11:44.841 [info] [KeybindingService]: / Received  keydown event - modifiers: [meta], code: MetaLeft, keyCode: 91, key: Meta
2024-11-04 05:11:44.841 [info] [KeybindingService]: | Converted keydown event - modifiers: [meta], code: MetaLeft, keyCode: 57 ('Meta')
2024-11-04 05:11:44.841 [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
2024-11-04 05:11:44.962 [info] [KeybindingService]: / Soft dispatching keyboard event
2024-11-04 05:11:44.962 [info] [KeybindingService]: | Resolving meta+[Enter]
2024-11-04 05:11:44.963 [info] [KeybindingService]: \ From 23 keybinding entries, no when clauses matched the context.
2024-11-04 05:11:44.977 [info] [KeybindingService]: / Received  keydown event - modifiers: [meta], code: Enter, keyCode: 13, key: Enter
2024-11-04 05:11:44.977 [info] [KeybindingService]: | Converted keydown event - modifiers: [meta], code: Enter, keyCode: 3 ('Enter')
2024-11-04 05:11:44.978 [info] [KeybindingService]: | Resolving meta+[Enter]
2024-11-04 05:11:44.978 [info] [KeybindingService]: \ From 23 keybinding entries, no when clauses matched the context.
2024-11-04 05:11:45.180 [info] [KeybindingService]: + Storing single modifier for possible chord meta.
2024-11-04 05:11:45.482 [info] [KeybindingService]: + Clearing single modifier due to 300ms elapsed.
```

alt+enter:

```
2024-11-04 05:12:01.051 [info] [KeybindingService]: / Soft dispatching keyboard event
2024-11-04 05:12:01.051 [info] [KeybindingService]: \ Keyboard event cannot be dispatched
2024-11-04 05:12:01.052 [info] [KeybindingService]: / Received  keydown event - modifiers: [alt], code: AltLeft, keyCode: 18, key: Alt
2024-11-04 05:12:01.052 [info] [KeybindingService]: | Converted keydown event - modifiers: [alt], code: AltLeft, keyCode: 6 ('Alt')
2024-11-04 05:12:01.052 [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
2024-11-04 05:12:01.156 [info] [KeybindingService]: / Soft dispatching keyboard event
2024-11-04 05:12:01.170 [info] [KeybindingService]: | Resolving alt+[Enter]
2024-11-04 05:12:01.170 [info] [KeybindingService]: \ From 8 keybinding entries, no when clauses matched the context.
2024-11-04 05:12:01.170 [info] [KeybindingService]: / Received  keydown event - modifiers: [alt], code: Enter, keyCode: 13, key: Enter
2024-11-04 05:12:01.170 [info] [KeybindingService]: | Converted keydown event - modifiers: [alt], code: Enter, keyCode: 3 ('Enter')
2024-11-04 05:12:01.170 [info] [KeybindingService]: | Resolving alt+[Enter]
2024-11-04 05:12:01.170 [info] [KeybindingService]: \ From 8 keybinding entries, no when clauses matched the context.
2024-11-04 05:12:01.358 [info] [KeybindingService]: + Ignoring single modifier alt due to it being pressed together with other keys.
```

### LINKED ISSUE #233305: Focusing terminal chat shows icons
After https://github.com/microsoft/vscode/pull/233207

Image

### LINKED ISSUE #233308: Terminal chat: Bring back buttons at bottom
This broke at some point.

### PR DESCRIPTION
**Pull request by Tyriar**
Fixes https://github.com/microsoft/vscode/issues/233308
Fixes https://github.com/microsoft/vscode/issues/232966
Fixes https://github.com/microsoft/vscode/issues/233305

### CODE REVIEW

File: src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatActions.ts [hunk 1]
```diff
@@ -233,22 +213,18 @@ registerActiveXtermAction({
...
+	category: AbstractInlineChatAction.category,
 	precondition: ContextKeyExpr.and(
 		ContextKeyExpr.or(TerminalContextKeys.processSupported, TerminalContextKeys.terminalHasBeenCreated),
 		TerminalChatContextKeys.requestActive.negate(),
 	),
 	icon: Codicon.commentDiscussion,
 	menu: [{
 		id: MENU_TERMINAL_CHAT_WIDGET_STATUS,
-		group: '0_main',
+		group: 'zzz',
```
Comment (meganrogge) [inline src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatActions.ts:224]: group name ?

File: src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatActions.ts [hunk 2]
```diff
@@ -233,22 +213,18 @@ registerActiveXtermAction({
...
 		order: 1,
+		isHiddenByDefault: true,
```
Comment (meganrogge) [inline src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatActions.ts:226]: should other actions, like `insertFirstCommand` and `run` also be hidden by default?

File: src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatActions.ts (hunk 1 above)
Comment (Tyriar) [reply src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatActions.ts:224]: This was actually copied directly from inline chat. Think it's that mainly to be last.

**Review by meganrogge — APPROVED** [review]
Works very well! Thanks for fixing this.

**Pull request merged**